src/cpuinfo_x86.c

```diff
@@ -145,12 +145,16 @@ static void ParseCpuId(const uint32_t max_cpuid_leaf, X86Info* info) {
 
   features->smx = IsBitSet(leaf_1.ecx, 6);
   features->cx16 = IsBitSet(leaf_1.ecx, 13);
+  features->movbe = IsBitSet(leaf_1.ecx, 22);
+  features->popcnt = IsBitSet(leaf_1.ecx, 23);
+  features->rdrnd = IsBitSet(leaf_1.ecx, 30);
```

Comment:
rdrand should be after f16c to write them in order

Reply:
Done